Plots/Panels/NCL_panel_35.py

```diff
@@ -0,0 +1,240 @@
+        if (temp >= -20.):
+          out_array[i,j] = out_array[i,j] +    \
+             midpt*tmp_array[2,k]*math.exp(temp)
+      dmin = min(dmin,out_array[i,j])
+      dmax = max(dmax,out_array[i,j])
+                   
+  out_array = (((out_array-dmin)/(dmax-dmin))*(maxv-minv))+minv
+
+  del tmp_array
+
+  return np.transpose(out_array,[1,0])
+
+
+def _get_double(obj,name):
+  return(NhlGetDouble(_int_id(obj),name))
+
+def _get_double_array(obj,name):
+  return(NhlGetDoubleArray(_int_id(obj),name))
+
+###############################################################################
+# Create dummy data
+nx = 100
+ny = 100
+data1 = generate_2d_array((ny,nx), 10, 10, -19., 16., 0)
+data2 = generate_2d_array((ny,nx), 10, 10, -28., 15., 1)
+data3 = generate_2d_array((ny,nx), 10, 10, -25., 18., 2)
+
+###############################################################################
+# Create figure and axes using gvutil
+fig, axs = plt.subplots(1, 3, figsize=(12, 6), sharex='all', sharey='all', gridspec_kw={'wspace': 0})
+
+gvutil.set_axes_limits_and_ticks(axs[0], xticks=np.arange(0, 120, 20), yticks=np.arange(0, 120, 20), xticklabels=np.arange(0, 100, 20), yticklabels=np.arange(0, 100, 20))
+gvutil.add_major_minor_ticks(axs[0], x_minor_per_major=4, y_minor_per_major=4)
+axs[0].tick_params(axis='both', which='both', top=True, left=True, right=False)
+axs[0].set_aspect(aspect='equal')
+
+gvutil.set_axes_limits_and_ticks(axs[1], xticks=np.arange(0, 120, 20), yticks=np.arange(0, 120, 20), xticklabels=np.arange(0, 100, 20), yticklabels=np.arange(0, 100, 20))
+gvutil.add_major_minor_ticks(axs[1], x_minor_per_major=4, y_minor_per_major=4)
+axs[1].tick_params(axis='both', which='both', top=True, left=False, right=False)
+axs[1].set_aspect(aspect='equal')
+
+gvutil.set_axes_limits_and_ticks(axs[2], xticks=np.arange(0, 120, 20), yticks=np.arange(0, 120, 20), xticklabels=np.arange(0, 100, 20), yticklabels=np.arange(0, 100, 20))
+gvutil.add_major_minor_ticks(axs[2], x_minor_per_major=4, y_minor_per_major=4)
+axs[2].tick_params(axis='both', which='both', top=True, left=False, right=True)
+axs[2].set_aspect(aspect='equal')
+
+###############################################################################
+# Plot data and create colorbar
+newcmap = gvcmaps.BlueYellowRed
+contour_levels = np.arange(-32,24,4) # levels=contour_levels ensures that each plot has the same scale
+
+filled1 = axs[0].contourf(data1, cmap=newcmap, levels=contour_levels)
+axs[0].contour(filled1, colors='k', linestyles='solid', linewidths=0.5)
+filled2 = axs[1].contourf(data2, cmap=newcmap, levels=contour_levels)
+axs[1].contour(filled2, colors='k', linestyles='solid', linewidths=0.5)
+filled3 = axs[2].contourf(data3, cmap=newcmap, levels=contour_levels)
+axs[2].contour(filled3, colors='k', linestyles='solid', linewidths=0.5)
+
+cbar = plt.colorbar(filled3, orientation='horizontal', ax=axs, ticks=np.arange(-28,20,4), shrink=0.75, drawedges=True, pad=0.1)
```

Comment:
Since `cbar` variable is not used later, we can directly call `plt.colorbar()` without assigning its output to `cbar`.